ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -777,6 +783,16 @@ default List<Partition> getPartitionsByExpr(org.apache.hadoop.hive.ql.metadata.T
             "for a table.");
   }
 
+  default List<Partition> getPartitions(org.apache.hadoop.hive.ql.metadata.Table table, 
+      Map<String, String> partitionSpec) throws SemanticException {
+    throw new UnsupportedOperationException("Storage handler does not support getting partitions for a table.");
+  }
+
+  default Partition getPartition(org.apache.hadoop.hive.ql.metadata.Table table, Map<String, String> partSpec) 
```

Comment:
`partitionSpec`
please keep param names consistent, that is an interface

Reply:
Done